tasks/makeSearchIndex.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs')
+const utils = require('./utils/utils')
+const {parseProjects} = require('./utils/parseProjects')
+
+if (require.main !== module) {
+  module.exports = function(config) {
+    const targets = ['build/.temp/search_index.json']
+    const boards = parseProjects(config, process.env.CACHED_BUILD)
```

Comment:
I didn't change this from the previous state, though I skeptical wether the right env var is `CACHED_BUILD` or `cached_build`

Reply:
What do you mean by previous state? It's definitely `CACHED_BUILD`.

Reply:
Oh, I think the `CACHED_BUILD` is refering the `var` in `.travis.yml` and it's gone now so it should use `processArgs` instead.

Reply:
We also set it and unset it on Travis settings: https://travis-ci.org/github/kitspace/kitspace/settings

Reply:
It fails [now](https://travis-ci.org/github/kitspace/kitspace/builds/728945333#L612) for not enough arguments to the configure command!

Reply:
Just use `process.env`? I don't understand why you changed that...